### PR TITLE
Added documentation for creating API by importing schema

### PR DESCRIPTION
**Pull request by insomniac12**

### CODE REVIEW

**Comment by sankalp0o** [comment]
Waiting to add other changes for the same spec

File: v6/postman/collections/data_formats.md [hunk 1]
```diff
@@ -70,49 +70,45 @@ A few commands which can be imported include:
 | `curl -X PUT --data-binary hello http://postman-echo.com/put`                                | Creates a POST request with raw data                                 |
 | `curl -X PUT --data-ascii 'a=b&c=d' http://postman-echo.com/put -H 'AccessToken:1234'`       | Creates a PUT request with urlencoded form data, and a custom header |
 
-## Importing RAML
+## Importing RAML/OpenAPI schema as collection or an API
 
-### Saving a RAML folder as a collection
+1. Click on the Import button.
+ [![import button](https://assets.postman.com/postman-docs/WS-collections-view-raml-1a.png)](https://assets.postman.com/postman-docs/WS-collections-view-raml-1a.png)
```
Comment (kaustavdm) [inline v6/postman/collections/data_formats.md]: There should be newlines between these. You'd want to use Markdown linter to sort these issues.

File: v6/postman/collections/data_formats.md [hunk 2]
```diff
@@ -70,49 +70,45 @@ A few commands which can be imported include:
...
+ Following modal shows up :
+  [![import sample](https://assets.postman.com/postman-docs/import+modal.png)](https://assets.postman.com/postman-docs/import+modal.png)
 
-1. Clone the repository containing the RAML definition to your local machine, or save it locally as a folder.
+2. Choose to import schema by file, folder, url or by copying as raw text.
+    > If multiple files are selected then schema can be imported only as [collection](/docs/postman/collections/intro_to_collections) and not as [APIs](/docs/postman/design_and_develop_apis/introduction_to_apis/) in Postman.
```
Comment (kaustavdm) [inline v6/postman/collections/data_formats.md]: This should not be a quote. This should be a paragraph instead.

File: v6/postman/collections/data_formats.md [hunk 3]
```diff
@@ -70,49 +70,45 @@ A few commands which can be imported include:
...
-[![confirmation message](https://assets.postman.com/postman-docs/WS-data-format-raml-2a.png)](https://assets.postman.com/postman-docs/WS-data-format-raml-2a.png)
-
-### OpenAPI files
-
-[OpenAPI 2.0](https://github.com/OAI/OpenAPI-Specification/tree/master/examples/v2.0)
-
-[OpenAPI 1.2](https://github.com/OAI/OpenAPI-Specification/wiki/Hello-World-Sample)
+**OpenAPI 2.0**
+Download an example OpenAPI 2.0 file:
+[petstore.yaml](https://github.com/OAI/OpenAPI-Specification/blob/master/examples/v2.0/yaml/petstore.yaml)
```
Comment (kaustavdm) [inline v6/postman/collections/data_formats.md]: - Introduce newlines between paragraphs
- Use a list to list down all the sample files.

File: v6/postman/collections/data_formats.md [hunk 4]
```diff
@@ -70,49 +70,45 @@ A few commands which can be imported include:
...
+[petstore.json]
+(https://github.com/OAI/OpenAPI-Specification/blob/master/examples/v2.0/json/petstore.json)  
+
+**OpenAPI 3.0**
+Download an example OpenAPI 3.0 file:
+[api-with-examples.yaml](https://github.com/OAI/OpenAPI-Specification/blob/master/examples/v3.0/api-with-examples.yaml)
+[petstore.yaml]
+(https://github.com/OAI/OpenAPI-Specification/blob/master/examples/v3.0/petstore.yaml)
```
Comment (kaustavdm) [inline v6/postman/collections/data_formats.md]: Broken markdown.

File: v6/postman/launching_postman/newbutton.md [hunk 5]
```diff
@@ -150,6 +151,18 @@ In the **Next steps** tab, see a list of suggested next steps to get the most ou
 
 [![next mock](https://assets.postman.com/postman-docs/WS-mock-next-steps.png)](https://assets.postman.com/postman-docs/WS-mock-next-steps.png)
 
+## Create an API
+
+An API allows you to manage all aspects of API design, development, and testing from within Postman. You can create an API only when you are signed in and online.
```
Comment (kaustavdm) [inline v6/postman/launching_postman/newbutton.md]: Last sentence can use a rewrite to make it sound more pleasant.

File: v6/postman/collections/data_formats.md [hunk 6]
```diff
@@ -70,49 +70,45 @@ A few commands which can be imported include:
 | `curl -X PUT --data-binary hello http://postman-echo.com/put`                                | Creates a POST request with raw data                                 |
 | `curl -X PUT --data-ascii 'a=b&c=d' http://postman-echo.com/put -H 'AccessToken:1234'`       | Creates a PUT request with urlencoded form data, and a custom header |
 
-## Importing RAML
+## Importing RAML/OpenAPI schema as collection or an API
```
[the remark this answers is not in the document]
Comment (sankalp0o) [reply v6/postman/collections/data_formats.md]: Let's just call this "Importing RAML/OpenAPI schema"

**Comment by sankalp0o** [comment]
@kaustavdm All the comments have been resolved, listing issues fixed and screenshots added. Could you do a final check and merge if no further changes are needed?

**Pull request merged**